Adyen.Test/BaseTest.cs

```diff
@@ -320,7 +320,7 @@ protected Client CreateAsyncMockTestClientApiKeyBasedRequest(string fileName)
         /// <returns>IClient implementation</returns>
         protected Client CreateMockTestClientPosCloudApiRequest(string fileName)
         {
-            var config = new Config { Endpoint = ClientConfig.CloudApiEndPointTest };
+            var config = new Config { Endpoint = ClientConfig.CloudApiEndPointTestSync };

```

Comment:
shouldn't `Config.CloudApiEndPoint` be used instead?
`Config.Endpoint` is the one for classic integration(pal)?
```suggestion
            var config = new Config { CloudApiEndPoint = ClientConfig.CloudApiEndPointTestSync };

```